internal/rest/k8s_utils.go

```diff
@@ -147,6 +147,8 @@ func (rest *RestOperations) SyncObjectStatuses() {
 			status.UpdateL4LBStatus(allServiceLBUpdateOptions, true)
 		}
 	}
-	utils.AviLog.Infof("Status syncing completed")
-	lib.AKOControlConfig().PodEventf(v1.EventTypeNormal, lib.StatusSync, "Status syncing completed")
+	if !lib.DisableSync {
```

Comment:
This seems to just control the log, but the sync is still happening. I think it's fine to log these since we are syncing statuses of already synced VSes in Avi

Reply:
addressed